Purchases/Networking/Operations/Handling/UserInfoAttributeParser.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  UserInfoAttributeParser.swift
+//
+//  Created by Joshua Liebowitz on 11/19/21.
+
+import Foundation
+
+class UserInfoAttributeParser {
+
+    func attributesUserInfoFromResponse(response: [String: Any], statusCode: Int) -> [String: Any] {
+        var resultDict: [String: Any] = [:]
+        let isInternalServerError = statusCode >= HTTPStatusCodes.internalServerError.rawValue
```

Comment:
Follow up to my comment above, this is another example, ideally this method is only called for a `200`, so error handling is done outside by `HTTPClient`.